2.6/test/run

```diff
@@ -182,5 +182,20 @@ for server in ${WEB_SERVERS[@]}; do
   info "All tests for the ${server}-test-app finished successfully."
   cleanup ${server}
 done
-
+pushd ${test_dir}
+  info "Check building using a Dockerfile"
+  ct_test_app_dockerfile ../../examples/from-dockerfile/2.6/Dockerfile 'https://github.com/sclorg/rails-ex.git' 'Welcome to your Rails application on OpenShift' app-src
+  t1=$?
+  if [ "$t1" == "0" ]; then
+                echo "test 1 from_dockerfile passed";
+        fi;
+  check_result $t1
+  info "Check building using a Dockerfile.s2i"
+  ct_test_app_dockerfile ../../examples/from-dockerfile/2.6/Dockerfile.s2i 'https://github.com/sclorg/rails-ex.git' 'Welcome to your Rails application on OpenShift' app-src
```

Comment:
Please wrap this in some loop, as we do for [WEB_SERVERS](https://github.com/sclorg/s2i-ruby-container/blob/72669de0cb8a6881118806f6a02636deaaa25311/2.5/test/run#L11) [here](https://github.com/sclorg/s2i-ruby-container/blob/72669de0cb8a6881118806f6a02636deaaa25311/2.5/test/run#L148).